docs/manual/introduction.tex

```diff
@@ -714,6 +714,8 @@
   These annotations may reduce the number of type-checking
   errors when running type-checking in the future; see
   Section~\ref{whole-program-inference}.
+  Using \<-Ainfer> produces \<.jaif> files.
+  Using \<-Ainfer=stubs> produces \<.astub> files.
```

Comment:
I would use singular "stub" instead of "stubs" here (or else the above will need to be `-Ainfer=jaifs`.

Reply:
Let's use `-Ainfer=jaifs` to avoid needing to change everything that relies on `-Ainfer=stubs`.